ci/inspect_changes.sh

```diff
@@ -37,6 +38,7 @@ declare -A dependencies=(
   [thrust]="cccl libcudacxx cub"
   [cudax]="cccl libcudacxx"
   [pycuda]="cccl libcudacxx cub thrust cudax"
```

Comment:
`cuda.parallel` depends on the C library:
```suggestion
  [pycuda]="cccl libcudacxx cub thrust c"
```